input/postgres/relations.go

```diff
@@ -11,6 +11,9 @@ import (
 	"github.com/pganalyze/collector/state"
 )
 
+const relationSQLdefaultSystemCatalogFilter = "n.nspname NOT IN ('pg_catalog', 'pg_toast', 'information_schema')"
```

Comment:
Note that "pg_toast" wasn't in all the previous queries (some only excluded pg_catalog and information_schema), but I don't see harm in excluding this always.

The one exception I can think of today where we actively look at TOAST data is for vacuums, and this change shouldn't have any effect on that.